### DISCUSSION TITLE
Engage not working

### DISCUSSION THREAD

**Discussion by martinmacecek-rearc** [noop-inc/documentation#132] [Questions] [answered]
The `Engage` button does not do anything for me:
<img width="1274" alt="Screenshot 2024-02-28 at 3 41 32 PM" src="https://github.com/noop-inc/documentation/assets/156843182/37cace1b-f963-4bef-9abc-23d171cd5d54">

I am not sure what is supposed to happen, but clicking it does not yield any result or action. This is on `0.9.0-arm64` after a system restart (to mitigate the workshop not starting).

Can anyone chime in what is supposed to happen and how to analyze the issue (if one at all)?

**Comment by 1800joe** [comment]
What should happen is a panel pops up with support, community, documentation and other resources.

We'll investigate what's going on. Thanks!

<img width="529" alt="Screenshot 2024-02-29 at 9 07 44 AM" src="https://github.com/noop-inc/documentation/assets/3111670/f2860622-c632-4356-b8b4-43996332fdf5">

**Reply by martinmacecek-rearc** [reply]
Just FYI: same problem still persists with version `0.9.1`, which was auto-updated today.

**Comment by arbitrarytech** [comment]
@martinmacecek-rearc Is it possible you have any kind of ad blocker on your computer or network that could be causing the issue?

**Reply by martinmacecek-rearc** [reply]
I first tried with my `pihole` deactivated, but no success with just that.
I then connected to my iPhone hotspot and that worked, surprisingly...

I don't have any other network/computer ad blockers running. Unless MacOS has one that I am not aware off. I do block `graph.facebook.com` though...
What URL(s) is(are) contacted when one clicks on `Engage`?

**Reply by martinmacecek-rearc** [reply, marked as answer]
Excluded all blacklisted URLs in my network, but that did not help in my case.

I think (unless there are more details around what URLs are contacted), we can close this discussion with the root cause being something on my network (probably router with some sort of firewall rule) blocking the `Engage` functionality as switching to a mobile hot spot made it work.
